great-idea/css/index.css

```diff
@@ -63,4 +63,260 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.container{
+	width: 90%;
+	margin-left: 50px;
+	display: flex;
+}
+
+.nav li {
+	margin: 55px 20px 10px 80px;
+	padding: 10px;
+	color: #c1c1c1;
+	cursor: pointer;
+}
+
+ul li a {
+	text-decoration: none;
+	color: inherit;
+}
+
+.active {
+	color: black;
+	font-weight: 900;
+}
+
+.nav ul {
+	display: flex; 
+	margin-bottom: 50px;
+}
+
+nav ul li a:hover {
+	color: black;
+}
+
+.button {
+	border-radius: 0;
+	border: 1px solid black;
+	font-size: 20px;
+	width: 200px;
+}
+
+.text-center {
+	text-align: center;
+}
+
+
+.header-text-container {
+	width: 60%;
+	text-align: center;
+	margin-bottom: 50px;
+}
+
+.header {
+	display: flex;
+	justify-content: space-between;	
+	margin: 0px 10% 0px 10%;
+}
+
+.header-text{
+	text-align: center;
+	font-size: 90px;
+}
+
+.new-line{
+	display: block;
+}
+
+.section-1 {
+	text-align: center;
+}
+
+.services-header-img {
+	width: 100%;
+}
+
+.products-info-vid {
+	width: 500px;
+	height: 350px;
+}
+
+.middle-img{
+	width: 90%;
+	justify-content: center;
+}
+
+.section-2 {
+	margin: 50px;
+}
+
+
+p {
+	text-align: justify;
+  text-justify: inter-word;
+}
+
+hr {
+	border: 1px solid black;
+}
+
+.h1-text {
+	font-size: 25px;
+}
+
+.footer {
+	margin-bottom: 50px;
+}
+
+.card-group {
+	flex-wrap: wrap;
+	justify-content: center;
+	border-top: 1px solid black;
+	border-bottom: 1px solid black;
+
+}
+
+.card {
+	border: 1px solid black;
+	margin: 3% 5% 3% 5%;
+	width: 30%;
+	background-color: #f4f3f4;
+}
+
+.card-body {
+	padding: 5%;
+	margin: 2%;
+}
+
+.card-img-container {
+	align-items: center;
+	text-align: center;
+}
+
+.card-img {
+	max-width: 100%;
+}
+
+.p-text {
+	font-size: 100%;
+}
+
+.services-button {
+	margin-top: 5%;
+	border: 1px solid black;
+	border-radius: 7px;
+	width: 45%;
+	height: 30%;
+	font-size: 20px;
+	background-color: #ffffff;
+}
+
+.services-button-text {
+	margin: 2%;
+}
+
+.square {
+	list-style-type: square;
+	line-height: 20px;
+}
+
+@media only screen and (max-width: 1370px) {
```

Comment:
It's great that you figured how to use media queries to your project. Well Done!
Under a certain breakpoint (at around 885px) a nasty horizontal scroll bar appears. Try to fix it!